frontend/st-marys-island-explorer/index.html

```
@@ -0,0 +1,351 @@
+        </p>
+        <div class="smi-image-grid">
+          <div class="smi-image-item"><img src="https://images.unsplash.com/photo-1582719428252-bd1a6an446ba?w=800&q=80" alt="St. Mary's Island basalt formations" loading="lazy"></div>
+          <div class="smi-image-item"><img src="https://images.unsplash.com/photo-1559495967-9729e07494ba?w=800&q=80" alt="White sand beach with basalt columns" loading="lazy"></div>
+          <div class="smi-image-item"><img src="https://images.unsplash.com/photo-1576091160399-16eabe2f69ef?w=800&q=80" alt="Hexagonal basalt columns at low tide" loading="lazy"></div>
+          <div class="smi-image-item"><img src="https://images.unsplash.com/photo-1583420601295-4e904370d6e5?w=800&q=80" alt="Turquoise water around basalt rocks" loading="lazy"></div>
+        </div>
+        <div class="smi-image-grid">
+          <div class="smi-image-item"><img src="https://images.unsplash.com/photo-1582696240815-27c64f1f871e?w=800&q=80" alt="Coconut palms on the shoreline" loading="lazy"></div>
+          <div class="smi-image-item"><img src="https://images.unsplash.com/photo-1574356629400-fd8a72411aa0?w=800&q=80" alt="Sunset over St. Mary's Island" loading="lazy"></div>
+          <div class="smi-image-item"><img src="https://images.unsplash.com/photo-1564507592333-c6065734d399?w=800&q=80" alt="Basalt columns reflected in tide pool" loading="lazy"></div>
+          <div class="smi-image-item"><img src="https://images.unsplash.com/photo-1582681245415-4651d830f067?w=800&q=80" alt "Cliff-side view of basalt formations" loading="lazy"></div>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)